params/version.go

```diff
@@ -21,10 +21,10 @@ import (
 )
 
 const (
-	VersionMajor = 1          // Major version component of the current release
-	VersionMinor = 10         // Minor version component of the current release
-	VersionPatch = 1          // Patch version component of the current release
-	VersionMeta  = "unstable" // Version metadata to append to the version string
+	VersionMajor = 1        // Major version component of the current release
+	VersionMinor = 10       // Minor version component of the current release
+	VersionPatch = 1        // Patch version component of the current release
+	VersionMeta  = "stable" // Version metadata to append to the version string
```

Comment:
Not a blocker, but is there any value to reporting a custom `clientVersion` from this patch? I'd have to do my homework on the implications, but this is technically not the same code as whatever `web3_clientVersion` reports. Probably not a high priority, but comes to mind as something to consider.

Reply:
(It might even already report something unique based on the way geth constructs this attribute. Worth a quick look.)